contracts/contracts/strategies/aerodrome/AerodromeAMOStrategy.sol

```diff
@@ -0,0 +1,959 @@
+        int24 _tickClosestToParity
+    ) InitializableAbstractStrategy(_stratConfig) {
+        WETH = _wethAddress;
+        OETHb = _oethbAddress;
+        swapRouter = ISwapRouter(_swapRouter);
+        positionManager = INonfungiblePositionManager(
+            _nonfungiblePositionManager
+        );
+        clPool = ICLPool(_clPool);
+        clGauge = ICLGauge(_clGauge);
+        helper = ISugarHelper(_sugarHelper);
+        sqrtRatioX96Tick0 = ISugarHelper(_sugarHelper).getSqrtRatioAtTick(
+            _lowerBoundingTick
+        );
+        sqrtRatioX96Tick1 = ISugarHelper(_sugarHelper).getSqrtRatioAtTick(
+            _upperBoundingTick
+        );
+        sqrtRatioX96TickClosestToParity = ISugarHelper(_sugarHelper)
+            .getSqrtRatioAtTick(_tickClosestToParity);
+
+        lowerTick = _lowerBoundingTick;
+        upperTick = _upperBoundingTick;
+        tickSpacing = 1;
+
+        require(
+            ICLPool(_clPool).token0() == _wethAddress,
+            "Only WETH supported as token0"
+        );
+        require(
+            ICLPool(_clPool).token1() == _oethbAddress,
+            "Only OETHb supported as token1"
+        );
+    }
+
+    /**
+     * @notice initialize function, to set up initial internal state
+     * @param _rewardTokenAddresses Address of reward token for platform
+     */
+    function initialize(
+        address[] memory _rewardTokenAddresses,
+        address[] memory,
+        address[] memory
+    ) external onlyGovernor initializer {
+        InitializableAbstractStrategy._initialize(
+            _rewardTokenAddresses,
+            // these should all be empty
+            new address[](0),
+            new address[](0)
+        );
+    }
+
+    /***************************************
+                  Configuration 
+    ****************************************/
+
+    /**
+     * @notice Set the new desired WETH share
+     * @param _amount The new amount specified in basis points
+     */
+    function setPoolWethShare(uint256 _amount) external onlyGovernor {
+        require(_amount < 1e18, "Invalid poolWethShare amount");
+        require(_amount > 0, "Invalid poolWethShare amount");
+
+        poolWethShare = _amount;
+        emit PoolWethShareUpdated(_amount);
+    }
+
+    /**
+     * @notice Specifies the amount of liquidity that is to be removed when
+     *         a rebalancing happens.
+     * @param _amount The new amount specified in basis points
+     */
+    function setWithdrawLiquidityShare(uint128 _amount) external onlyGovernor {
+        require(_amount < 1e18, "Invalid withdrawLiquidityShare amount");
+
+        withdrawLiquidityShare = _amount;
+        emit WithdrawLiqiudityShareUpdated(_amount);
+    }
+
+    /**
+     * @notice Specifies how the target WETH share of the pool defined by the `poolWethShare` can
+     *         vary from the configured value after rebalancing.
+     * @param _amount The new amount specified in basis points
```

Comment:
This is no longer in basis points

Reply:
tnx fixed here: https://github.com/OriginProtocol/origin-dollar/pull/2184/commits/bea826f40aba9b51e871a9b2865e28ed1afd8ef8